--- enable-observed.factor
+++ enable-timed.factor
@@ -5,8 +5,6 @@
 compiler.cfg.register-allocation.chordal compiler.cfg.register-allocation.chordal.recoloring
 compiler.cfg.registers kernel locals math namespaces sequences ;
 IN: compiler.cfg.register-allocation.chordal
-SYMBOL: experimental-recolor-counts
-V{ } clone experimental-recolor-counts set-global
 :: assign-certified-colors ( intervals cfg available -- )
     intervals interference-graph :> graph
     graph maximum-cardinality-order :> order
@@ -14,7 +12,6 @@
     cfg graph weighted-ssa-affinities :> weighted
     graph order weighted available preference-guided-colors :> colors
     graph order weighted available colors improve-weighted-colors
-    dup experimental-recolor-counts get push
     "recolored-vertices" chordal-statistics get set-at
     colors graph-colors namespaces:set
     intervals [| interval |
